sw/host/opentitanlib/src/util/mod.rs

```diff
@@ -54,6 +55,18 @@ macro_rules! testdata {
     }};
 }
 
+/// Create a filename in a temporary directory.
+///
+/// When running under bazel, the filename will exist in the test's undeclared outputs directory.
+pub fn tmpfilename(name: &str) -> String {
```

Comment:
FYI we already have [tempfile](https://crates.io/crates/tempfile) in our dependency tree in case you want to use that instead

Reply:
I want the files to be located in `TEST_UNDECLARED_OUTPUTS_DIR` if that is defined by the test execution environment.  Those files get kept after bazel test execution, so if there is a problem, they can be inspected.

Reply:
That sounds sensible, thanks